lib/config/presets/internal/monorepo.ts

```diff
@@ -223,6 +223,12 @@ const repoGroups = {
   surveyjs: 'https://github.com/surveyjs/surveyjs',
   'swashbuckle-aspnetcore':
     'https://github.com/domaindrivendev/Swashbuckle.AspNetCore',
+  swc: [
+    'https://github.com/swc-project/swc',
+    'https://github.com/swc-project/plugins',
+    'https://github.com/swc-project/jest',
+    'https://github.com/swc-project/cli',
```

Comment:
Other presets in this file have one URL in them. The current preset has four URLs. I'll let the maintainers decide which option is best.

### Option 1: group whole `swc-project` repository

```ts
'swc-project': 'https://github.com/swc-project',
```

### Option 2: separate entries for each repository

```ts
'swc-project/swc': 'https://github.com/swc-project/swc',
'swc-project/plugins': 'https://github.com/swc-project/plugins',
'swc-project/jest': 'https://github.com/swc-project/jest',
'swc-project/cli': 'https://github.com/swc-project/cli',
```

### Option 3: do nothing

Maybe this is a valid config, and I'm confusing myself somehow. In that case you can ignore me. 😄

Reply:
do they have more repos which should not grouped? then we stay as is, otherwise we can group by organization url

Reply:
Grouping by oraganization url seem to be a fine for me, so I refactored in [8760e3c](https://github.com/renovatebot/renovate/pull/21431/commits/8760e3c5251916a9426d8bf3c08930c99eec1b68).